src/app-host/app-host.js

```diff
@@ -72,22 +75,27 @@ socket.on('exec-failure', function (data) {
     }
 });
 
+socket.on('start-live-reload', function () {
+    livereload.start(socket);
+});
+
+telemetry.init(socket);
```

Comment:
Do we need to call `init()` every time? What if we don't want to enable telemetry?